games/xworld3d/xworld3d_simulator.cpp

```diff
@@ -84,32 +89,29 @@ class X3SimulatorImpl {
                       size_t img_height_out, size_t img_width_out,
                       bool color = false);
 
-    cv::Mat concat_images(cv::Mat img1, cv::Mat img2,
-                                           bool is_vertical);
+    cv::Mat concat_images(cv::Mat img1, cv::Mat img2, bool is_vertical);
```

Comment:
i copied this function from xworld. do we need to have a separate file for utility functions like this?